examples/amazon-linux-ec2-default-vpc-ico-by-densify/README.md

```diff
@@ -0,0 +1,104 @@
+<p align="center">
+  <img src="https://github.com/intel/terraform-intel-aws-vm/blob/main/images/logo-classicblue-800px.png?raw=true" alt="Intel Logo" width="250"/>
+</p>
+
+# Intel® Optimized Cloud Modules for Terraform
+
+© Copyright 2022, Intel Corporation
+
+## Terraform Intel AWS VM - Linux VM in Default VPC
+<p align="center">
+  <img src="https://github.com/intel/terraform-intel-aws-mysql/blob/main/images/aws-ec2-ico.png?raw=true" alt="Intel + Densify Logo" width="250"/>
+</p>
+
+This example creates creates AWS EC2 instance on Linux Operating System in the default VPC using recommended instance from Intel Cloud Optimizer by Densify. Intel® Cloud Optimizer is a collaboration between Densify and Intel targeted at getting you the most from your cloud investment. Intel Cloud Optimizer by Densify helps customers optimize their cloud investments and ensure optimal performance for every workload. Using this example requires a densify_recommndations.auto.tfvars file. You are expected to generate this file so this is a sample file only. In this sample file we will use example of recommended instance type of db.m6i.xlarge.
```

Comment:
Can we add more bullet points and links on how to: 
- Find out more 
- Contact Densify/Intel